src/main/java/io/jenkins/plugins/orka/OrkaCloud.java

```diff
@@ -144,39 +144,39 @@ public AgentTemplate getTemplate(Label label) {
 
     public List<OrkaVM> getVMs() throws IOException {
         return new OrkaClientProxyFactory()
-                .getOrkaClientProxy(this.endpoint, this.credentialsId, this.useJenkinsProxySettings, 
+                .getOrkaClientProxy(this.endpoint, this.credentialsId, this.useJenkinsProxySettings,
                         this.ignoreSSLErrors)
                 .getVMs();
     }
-    
+
     public List<OrkaVMConfig> getVMConfigs() throws IOException {
         return new OrkaClientProxyFactory()
-                .getOrkaClientProxy(this.endpoint, this.credentialsId, this.useJenkinsProxySettings, 
+                .getOrkaClientProxy(this.endpoint, this.credentialsId, this.useJenkinsProxySettings,
                         this.ignoreSSLErrors)
                 .getVMConfigs();
     }
 
     public ConfigurationResponse createConfiguration(String name, String image, String baseImage, String configTemplate,
```

Comment:
This is a breaking change.
We should introduce an overload for this method.

Reply:
Looks like that change was from main. I just rebased on main again, and it's no longer in the diff.

Reply:
Did need overloads there, so added.

Reply:
A space maybe? Is this even the line you were trying to highlight?

Reply:
Nevermind. You were talking about needing an overload for createConfiguration. Fixed